test/unit/org/apache/cassandra/db/ClusteringHeapSizeTest.java

```diff
@@ -64,6 +65,8 @@ public void unsharedHeapSizeExcludingDataLTEUnsharedHeapSize()
     @Test
     public void testSingletonClusteringHeapSize()
     {
+        if (this.clustering.accessor() == NativeAccessor.instance)
```

Comment:
Nit: change this to `Assume(this.clustering.accessor() != NativeAccessor.instance)`.

Reply:
fixed by using the recently introduced ensureAccessorFactorySupport